pkg/common/cns-lib/volume/manager.go

```diff
@@ -224,11 +224,16 @@ type createVolumeTaskDetails struct {
 func GetManager(ctx context.Context, vc *cnsvsphere.VirtualCenter,
 	operationStore cnsvolumeoperationrequest.VolumeOperationRequest,
 	idempotencyHandlingEnabled, multivCenterEnabled,
-	multivCenterTopologyDeployment bool,
+	multivCenterTopologyDeployment, isStorageQuotaM2FSSEnabled bool,
 	clusterFlavor cnstypes.CnsClusterFlavor) (Manager, error) {
 	log := logger.GetLogger(ctx)
 	managerInstanceLock.Lock()
 	defer managerInstanceLock.Unlock()
+	if isStorageQuotaM2FSSEnabled {
+		log.Infof("Setting version FCD_QUERY_RTINFO_SKIP_RUNTIME_IOSZ to vCenter: %q vim25 client, and cnsclient", vc.Config.Host)
+		vc.Client.Version = "FCD_QUERY_RTINFO_SKIP_RUNTIME_IOSZ"
+		vc.CnsClient.Version = vc.Client.Version
+	}
```

Comment:
Can this be moved to virtualcenter.go where we create the client? Maybe using isFSSEnabled?

Reply:
not sure we can use isFSSEnabled as virtualcenter.go is belong to package common and it will create cyclic dependency issue.
please let me know if you have any suggestion to achieve this avoiding cyclic dep issue